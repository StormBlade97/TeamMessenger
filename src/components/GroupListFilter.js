import React from 'react'

const GroupListFilter = (props) => {
    return (
        <div>
            search:
            <input onChange={props.onChange}/>
        </div>
    )
}

export default GroupListFilter