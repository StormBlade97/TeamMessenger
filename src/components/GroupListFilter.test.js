import React from 'react';
import GroupListFilter from './GroupListFilter';
import renderer from 'react-test-renderer';

describe('Component: GroupListFilter', () => {

    it('renders a snapshot', () => {

        function handleGroupListFilterChange() {

        }

        const tree = renderer.create(<GroupListFilter onChange={handleGroupListFilterChange()}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
})