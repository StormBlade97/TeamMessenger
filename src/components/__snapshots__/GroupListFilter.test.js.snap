exports[`Component: GroupListFilter renders a snapshot 1`] = `
<div>
  search:
  <input
    onChange={undefined} />
</div>
`;
